pkg/pulsar/topic.go

```diff
@@ -224,6 +225,15 @@ type Topics interface {
 
 	// SetInactiveTopicPolicies sets the inactive topic policies on a topic
 	SetInactiveTopicPolicies(topic utils.TopicName, data utils.InactiveTopicPolicies) error
+
+	// SetSchemaCompatibilityStrategy sets the strategy used to check the a new schema provided
+	// by a producer is compatible with the current schema before it is installed
+	SetSchemaCompatibilityStrategy(topic utils.TopicName,
+		strategy utils.SchemaCompatibilityStrategy) error
```

Comment:
You need to add a new structure `SchemaCompatibilityStrategyV2`, link to  https://github.com/apache/pulsar/blob/master/pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/SchemaCompatibilityStrategy.java#L24

Reply:
There are different values here.